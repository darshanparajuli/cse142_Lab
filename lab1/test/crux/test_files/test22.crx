or and not let var array true false if else while return
