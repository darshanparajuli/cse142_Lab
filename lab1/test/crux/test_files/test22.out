OR(lineNum:1, charPos:1)
AND(lineNum:1, charPos:4)
NOT(lineNum:1, charPos:8)
LET(lineNum:1, charPos:12)
VAR(lineNum:1, charPos:16)
ARRAY(lineNum:1, charPos:20)
TRUE(lineNum:1, charPos:26)
FALSE(lineNum:1, charPos:31)
IF(lineNum:1, charPos:37)
ELSE(lineNum:1, charPos:40)
WHILE(lineNum:1, charPos:45)
RETURN(lineNum:1, charPos:51)
EOF(lineNum:2, charPos:1)
